### PR TITLE
[KEP - 4191]: Use Cadvisor labels rather than hard coding them into kubelet

### PR DESCRIPTION
**Pull request by kannon92**
#### What type of PR is this?
We added a label for the read-only and writeable layers for crio on cadvisor side.

This PR uses the cadvisor constants rather than hardcoding them into kubelet.

#### What this PR does / why we need it:
Use cadvisor constants rather than the ones in kubelet.
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by bart0sh** [comment]
@kannon92 please, run `hack/update-vendor.sh` to fix CI job `pull-kubernetes-dependencies`.

**Comment by SergeyKanzhelev** [comment]
this looks like WIP

/wip

**Comment by kannon92** [comment]
/hold

using this to test split disk e2e tests.

this PR should not be merged because I modify cadvisor directly. I need a release of it and then I can update this PR.

**Comment by kannon92** [comment]
@dims @bobbypage any timeline on a cadvisor release? I already have these changes in cadvisor. Just waiting for a release and then this should be good to go.

**Comment by dchen1107** [comment]
Discussed at SIG Node today. @bobbypage will do a release cut shortly to unblock the progress.

**Comment by kannon92** [comment]
/hold cancel
/assign @mrunalp 

With cadvisor release, this is ready to go.

**Review by mrunalp — APPROVED** [review]
/lgtm

**Pull request merged**
